### ISSUE TITLE
cu118 cublas error solution  CUDA11报错cublas解决方案

### ISSUE THREAD

**Issue by DoItEric** [jianchang512/pyvideotrans#481]
@jianchang512 

关于cuda11 报错 cublas的解决方案：

1、报错的原因是因为fast模式使用的 fast-whisper 主要支持CUDA12，对于CUDA11需要进行依赖包的更换：

《引用自fast-whisper》：https://github.com/SYSTRAN/faster-whisper/tree/master
Note: Latest versions of ctranslate2 support CUDA 12 only. For CUDA 11, the current workaround is downgrading to the 3.24.0 version of ctranslate2 (This can be done with pip install --force-reinstall ctranslate2==3.24.0 or specifying the version in a requirements.txt).
★将ctranslate2的版本降级到3.24.0  change ctranslate version to 3.24.0 

2、根据错误信息调整numpy版本到合适的版本

3、还请题主更新requirment文件，区分cuda11的支持（或者我希望能成为项目贡献者，帮上忙【PS：我的时间不是很固定】）
我的pip list：

Package                        Version
------------------------------ ------------
absl-py                        2.0.0
aiohttp                        3.9.5
aiosignal                      1.3.1
altgraph                       0.17.4
annotated-types                0.6.0
anyio                          3.7.1
asttokens                      2.4.1
astunparse                     1.6.3
async-timeout                  4.0.3
attrs                          23.1.0
audioread                      3.0.1
av                             11.0.0
azure-cognitiveservices-speech 1.37.0
cachetools                     5.3.2
certifi                        2024.2.2
cffi                           1.16.0
chardet                        3.0.4
charset-normalizer             3.2.0
click                          7.1.2
colorama                       0.4.6
coloredlogs                    15.0.1
contourpy                      1.1.1
ctranslate2                    3.24.0
cycler                         0.12.1
decorator                      4.4.2
deepl                          1.16.1
distro                         1.8.0
edge-tts                       6.1.11
elevenlabs                     0.2.27
exceptiongroup                 1.1.3
executing                      2.0.1
faster-whisper                 1.0.3
ffmpeg-python                  0.2.0
filelock                       3.12.4
flatbuffers                    1.12
fonttools                      4.43.1
frozenlist                     1.4.0
fsspec                         2023.10.0
future                         0.18.3
gast                           0.4.0
google-ai-generativelanguage   0.6.2
google-api-core                2.19.0
google-api-python-client       2.128.0
google-auth                    2.29.0
google-auth-httplib2           0.2.0
google-auth-oauthlib           0.4.6
google-generativeai            0.5.2
google-pasta                   0.2.0
googleapis-common-protos       1.63.0
grpcio                         1.60.0
grpcio-status                  1.60.0
gTTS                           2.5.1
h11                            0.14.0
h2                             3.2.0
h5py                           3.10.0
hpack                          3.0.0
hstspreload                    2023.1.1
httpcore                       1.0.2
httplib2                       0.22.0
httpx                          0.25.1
huggingface-hub                0.17.3
humanfriendly                  10.0
hyperframe                     5.2.0
idna                           2.10
imageio                        2.31.4
imageio-ffmpeg                 0.4.9
ipython                        8.23.0
jedi                           0.19.1
Jinja2                         3.1.2
joblib                         1.3.2
keras                          2.9.0
Keras-Preprocessing            1.1.2
kiwisolver                     1.4.5
lazy_loader                    0.3
libclang                       16.0.6
librosa                        0.10.1
llvmlite                       0.42.0
Markdown                       3.5
MarkupSafe                     2.1.3
matplotlib-inline              0.1.6
more-itertools                 10.1.0
moviepy                        1.0.3
mpmath                         1.3.0
msgpack                        1.0.7
multidict                      6.0.4
networkx                       3.2
norbert                        0.2.1
numba                          0.59.1
numpy                          1.26.0
oauthlib                       3.2.2
onnxruntime                    1.18.1
openai                         1.2.3
openai-whisper                 20231117
opt-einsum                     3.3.0
ordered-set                    4.1.0
packaging                      23.1
pandas                         1.5.3
parso                          0.8.3
pefile                         2023.2.7
Pillow                         10.0.1
pip                            24.0
platformdirs                   3.11.0
plyer                          2.1.0
pooch                          1.7.0
proglog                        0.1.10
prompt-toolkit                 3.0.43
proto-plus                     1.23.0
protobuf                       4.25.1
pure-eval                      0.2.2
pyasn1                         0.5.0
pyasn1-modules                 0.3.0
pycparser                      2.21
pydantic                       2.4.2
pydantic_core                  2.10.1
pydub                          0.25.1
pygame                         2.5.2
Pygments                       2.17.2
pyinstaller                    6.6.0
pyinstaller-hooks-contrib      2024.4
pyparsing                      3.1.1
pyreadline3                    3.4.1
PySide6                        6.7.0
PySide6_Addons                 6.7.0
PySide6_Essentials             6.7.0
PySoundFile                    0.9.0.post1
python-dateutil                2.8.2
pytz                           2023.3.post1
pywin32-ctypes                 0.2.2
pywinstyles                    1.4
PyYAML                         6.0.1
QDarkStyle                     3.2.3
QtPy                           2.4.1
regex                          2023.10.3
requests                       2.31.0
requests-oauthlib              1.3.1
resampy                        0.4.2
rfc3986                        1.5.0
rsa                            4.9
samplerate                     0.2.1
scikit-learn                   1.3.2
scipy                          1.11.3
setuptools                     71.1.0
shiboken6                      6.7.0
six                            1.16.0
sniffio                        1.3.0
soundfile                      0.12.1
soxr                           0.3.7
SpeechRecognition              3.10.0
srt                            3.5.2
stack-data                     0.6.3
sympy                          1.12
tencentcloud-sdk-python-common 3.0.1032
tencentcloud-sdk-python-tmt    3.0.1032
termcolor                      2.3.0
threadpoolctl                  3.2.0
tiktoken                       0.6.0
tokenizers                     0.15.2
torch                          2.4.0+cu118
torchaudio                     2.4.0+cu118
tqdm                           4.66.1
traitlets                      5.14.1
typer                          0.3.2
typing_extensions              4.8.0
uritemplate                    4.1.1
urllib3                        2.0.5
wcwidth                        0.2.12
websockets                     12.0
Werkzeug                       3.0.1
wheel                          0.43.0
wrapt                          1.15.0
yarl                           1.9.2
zhconv                         1.4.3
